backend/inspirehep/editor/views.py

```diff
@@ -117,9 +117,14 @@ def get_revisions(endpoint, pid_value):
         revisions = []
         for revision in reversed(record.revisions):
             transaction_id = revision.model.transaction_id
-
-            user = Transaction.query.filter(Transaction.id == transaction_id).one().user
-            user_email = user.email if user else "system"
+            transaction = Transaction.query.filter(
```

Comment:
no tests for that 😿

Reply:
Updated! The test added should cover the erroneous scenario !